examples/cifar10/cifar10-sim/README.md

```diff
@@ -0,0 +1,178 @@
+
+### 3.2 Centralized training
+
+To simulate a centralized training baseline, we run FL with 1 client for 25 local epochs but only for one round. 
+It takes circa 6 minutes on an NVIDIA TitanX GPU.
+```
+./set_alpha.sh cifar10_central 0.0
+nvflare simulator job_configs/cifar10_central --workspace ${RESULT_ROOT}/central --threads 1 --n_clients 1
+```
+Note, here `alpha=0.0` means that no heterogeneous data splits are being generated.
+
+You can visualize the training progress by running `tensorboard --logdir=${RESULT_ROOT}`
+![Central training curve](./figs/central_training.png)
+
+### 3.3 FedAvg on different data splits
+
+FedAvg (8 clients). Here we run for 50 rounds, with 4 local epochs. Corresponding roughly 
+to the same number of iterations across clients as in the central baseline above (50*4 divided by 8 clients is 25):
+Each job will take about 35 minutes, depending on your system. 
+
+You can copy the whole block into the terminal, and it will execute each experiment one after the other.
+```
+./set_alpha.sh cifar10_fedavg 1.0
+nvflare simulator job_configs/cifar10_fedavg --workspace ${RESULT_ROOT}/fedavg_alpha1.0 --threads 8 --n_clients 8
+./set_alpha.sh cifar10_fedavg 0.5
+nvflare simulator job_configs/cifar10_fedavg --workspace ${RESULT_ROOT}/fedavg_alpha0.5 --threads 8 --n_clients 8
+./set_alpha.sh cifar10_fedavg 0.3
+nvflare simulator job_configs/cifar10_fedavg --workspace ${RESULT_ROOT}/fedavg_alpha0.3 --threads 8 --n_clients 8
+./set_alpha.sh cifar10_fedavg 0.1
+nvflare simulator job_configs/cifar10_fedavg --workspace ${RESULT_ROOT}/fedavg_alpha0.1 --threads 8 --n_clients 8
+```
+
+### 3.4: Advanced FL algorithms (FedProx, FedOpt, and SCAFFOLD)
+
+Next, let's try some different FL algorithms on a more heterogeneous split:
+
+[FedProx](https://arxiv.org/abs/1812.06127) adds a regularizer to the loss used in `CIFAR10Learner` (`fedproxloss_mu`)`:
+```
+./set_alpha.sh cifar10_fedprox 0.1
+nvflare simulator job_configs/cifar10_fedprox --workspace ${RESULT_ROOT}/fedprox_alpha0.1 --threads 8 --n_clients 8
+```
+[FedOpt](https://arxiv.org/abs/2003.00295) uses a new ShareableGenerator to update the global model on the server using a PyTorch optimizer. 
+Here SGD with momentum and cosine learning rate decay:
+```
+./set_alpha.sh cifar10_fedopt 0.1
+nvflare simulator job_configs/cifar10_fedopt --workspace ${RESULT_ROOT}/fedopt_alpha0.1 --threads 8 --n_clients 8
+```
+[SCAFFOLD](https://arxiv.org/abs/1910.06378) uses a slightly modified version of the CIFAR-10 Learner implementation, namely the `CIFAR10ScaffoldLearner`, which adds a correction term during local training following the [implementation](https://github.com/Xtra-Computing/NIID-Bench) as described in [Li et al.](https://arxiv.org/abs/2102.02079)
+```
+./set_alpha.sh cifar10_scaffold 0.1
+nvflare simulator job_configs/cifar10_scaffold --workspace ${RESULT_ROOT}/scaffold_alpha0.1 --threads 8 --n_clients 8
+```
+
+### 4. Running experiments in parallel
```

Comment:
should this be `### 3.5 Running experiments in parallel`

Reply:
Right. changed.